app/models/v2/report.rb

```diff
@@ -78,5 +78,10 @@ def profile_title
     def all_systems_exposed
       total_system_count == try(:aggregate_assigned_system_count)
     end
+
+    def delete_test_results
+      V2::RuleResult.joins(test_result: :tailoring).where(tailorings: { policy_id: id }).delete_all
+      TestResult.where(id: test_results.select(:id)).delete_all
```

Comment:
Is this V1 reference? Ideally we should not do that, the point of the trigger/function should be to not use V1.